sequencer/src/api/sql.rs

```diff
@@ -629,12 +638,14 @@ async fn reward_header_dependencies<Mode: TransactionMode>(
     let epoch_height = instance.epoch_height;
 
     let Some(epoch_height) = epoch_height else {
-        bail!("epoch height not set");
+        tracing::warn!("epoch height is not set. returning empty reward_header_dependencies");
+        return Ok(HashSet::new());
```

Comment:
Is this acceptable? It won't lead to false positives? Under what scenario is the epoch height not set?

Reply:
so If we are running on version < EpochVersion, this function will be called anyway but it shouldn't error out I think

Reply:
epoch height is an Optoon in NodeState. I guess it doesn't matter because of the way it is set lol

Reply:
Oh I see. This is basically checking whether the 0.3 upgrade is configured at all (statically), and the later version check is further checking whether the upgrade has been enabled yet for each header?

Reply:
yes

Reply:
tho NodeState always passes Some(epoch_height) which is not okay, but will fix it later